pyproject.toml

```diff
@@ -28,9 +28,18 @@ maintainers = [
     { name = "Michael Götte", email = "michael.goette@helmholtz-berlin.de" },
 ]
 license = { file = "LICENSE" }
-#dependencies = ["nomad-lab>=1.2.2dev578"]
+dependencies = [
+    "nomad-lab>=1.2.2dev578",
```

Comment:
i think this is the same issue as in baseclass plugin and might not be working?

Reply:
The issue here is newer versions of `nomad-lab` was only released on our Gitlab pypi server. If you want to install it, you'll have to pass the extra index url. This is mentioned in the [readme](https://github.com/FAIRmat-NFDI/cookiecutter-nomad-plugin/tree/main/nomad-%7B%7Bcookiecutter.plugin_name%7D%7D#install) of cookiecutter plugins.

You can install it the following way: 
```sh
pip install -e '.[dev]' --index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple
```

Reply:
Can this be added to the Pytoml file?

Reply:
pip doesn't support that.

You could use [uv](https://github.com/astral-sh/uv) instead, which supports this feature.

```toml
[tool.uv.pip]
index-url = "https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple"
```

`uv pip install -e '.[dev]` will work then.